ELK_Listener/ci_listener.yaml

```diff
@@ -24,17 +24,17 @@
             import numbers
             import dateutil.parser as dp
 
+            # Print the environment to console log for debugging purposes
+            for key in os.environ.keys():
+                 print "%s=%s" % (key, os.environ[key])
```

Comment:
Shall we always use "print( )" syntax (python3 compatible)?
Didn't notice this earlier. Other print lines seem good.